docs/src/fibers.md

```diff
@@ -57,11 +57,48 @@ Dense [1:4]
 ├─[4]: 0.0
 
 julia> A_fbr[:, 3]
-Dense [1:4]
-├─[1]: 4.4
-├─[2]: 0.0
-├─[3]: 5.5
-├─[4]: 0.0
+ERROR: ArgumentError: invalid index: 1.0 of type Float64
```

Comment:
It looks like your implementation is mixing up value and index types here

Reply:
Fixed by doctest.